hw_16/js/script.js

```diff
@@ -0,0 +1,126 @@
+// 1. Дано масив з елементами різних типів.Створити функцію,
+// яка вираховує середнє арифметичне лише числових елементів даного масиву.
+
+const array = ['a', 6, 76, 4, 'fjf', undefined];
+
+const arrayNumbers = array.filter(ele => typeof ele === 'number');
+
+const average = (arr) => {
+    let sum = 0;
+    for (let i of arr) {
+        sum += i;
+    }
+    const mathAv = sum / arr.length;
+    return mathAv;
+}
+
+const resultAverage = average(arrayNumbers);
```

Comment:
it will be better to combine filter with loop to avoid redundant loop

Why you avoid array methods?

Reply:
Thank you! Changed the code using reduce method